costmap_converter/src/costmap_converter_node.cpp

```diff
@@ -119,10 +119,9 @@ class CostmapStandaloneConversion : public rclcpp::Node {
 
     if (converter_) {
       converter_->setOdomTopic(odom_topic);
-      converter_->initialize(
-          std::make_shared<rclcpp::Node>("intra_node", "costmap_converter"));
+      converter_->initialize(shared_from_this());
```

Comment:
we don't need to do this but I don't see the point in creating another node

Reply:
yes, also DDS is happy if there are fewer nodes